app/controllers/openid_connect/logout_controller.rb

```diff
@@ -28,6 +48,21 @@ def index
       end
     end
 
+    def handle_successful_logout_request(redirect_uri)
+      if logout_params[:client_id] && logout_params[:id_token_hint].nil? && current_user
```

Comment:
Do we have test coverage exercising the `current_user` handling? It took me a moment to figure out what we're doing here (I assume the intent is to redirect back to the SP immediately if the user was never logged in?).

Reply:
https://github.com/18F/identity-idp/blob/72d2714d81538723a590394e4dcc5adb8fb3d1d9/spec/controllers/openid_connect/logout_controller_spec.rb#L451-L457 and https://github.com/18F/identity-idp/blob/72d2714d81538723a590394e4dcc5adb8fb3d1d9/spec/controllers/openid_connect/logout_controller_spec.rb#L665-L671 should cover it

And yeah, the intent is to redirect back to the SP if there is no session to confirm logging out of